blob/parser.go

```diff
@@ -21,6 +21,8 @@ type parser struct {
 
 // set tries to find the first blob's share by skipping padding shares and
 // sets the metadata of the blob(index and length)
+//
+//nolint:unused
```

Comment:
@vgonkivs, should we keep this code if it's unused now?

Reply:
Actually, we are likely gonna use it later on, but on the server side. The only way for the server to serve blobs with proofs is by parsing the whole namespace/row and this code is gonna be useful there, so we might keep it for now